tools/build/Library/InstallUnpacker/index_template.php

```diff
@@ -26,52 +26,102 @@
 
 set_time_limit(0);
 
+define('_PS_INSTALL_MINIMUM_PHP_VERSION_ID_', 50400);
+define('_PS_INSTALL_MINIMUM_PHP_VERSION_', '5.4');
```

Comment:
As we also have this constants in the file `install-dev/install_version.php` (See https://github.com/PrestaShop/PrestaShop/pull/8843), we should improve the release generator from @alegout to make the update of these values possible.